apb.yml

```diff
@@ -27,7 +27,7 @@ plans:
         required: True
         title: Sync Application Name
       - name: SYNC_APP_DOCKER_IMAGE
-        description: URL of your Sync Application Docker Image, e.g. docker.io/aerogear/voyager-server-example-task
+        description: URL of your Sync Application Docker Image, e.g. docker.io/aerogear/voyager-server-example-basic
```

Comment:
Not sure about this...
@aliok  @wei-lee , do we want to offer here the server app that doesn't use DB configuration (`docker.io/aerogear/voyager-server-example-basic`) or the one that actually requires running db server (`docker.io/aerogear/voyager-server-example-task`).

Or maybe we could update the description and give instructions for both apps?

Reply:
I changed it without thinking in detail. I am ok to revert it.
@wei-lee @psturc

Reply:
+1 for keeping `voyager-server-example-task`

Reply:
+1 on keeping the old value

Reply:
reverted. merging now.